### PR TITLE
Add some util with typescript

### PR DESCRIPTION
**Pull request by boatnoodle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Easy ideas to start contributing on Github, get [free T-Shirts](http://hacktober
 
 - Add ideas to this repo! 😂
 - Fix code formatting.
-- Fix typos. (Try searching `funcation` in Github...)
+- Fix typos. (Try searching `function` in Github...)
```
Comment (narze) [inline README.md]: This is intended to be a typo

File: util/greeting-sentences.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const greetingSentences = (
```
Comment (narze) [inline util/greeting-sentences.ts:1]: Where would these scripts be used?

Comment (boatnoodle) [reply util/greeting-sentences.ts:1]: I just want to create these function for others can be used to make a landing page or something else Lol.

File: README.md (hunk 1 above)
Comment (boatnoodle) [reply README.md]: Oh, I see.

File: util/greeting-sentences.ts (hunk 2 above)
Comment (narze) [reply util/greeting-sentences.ts:1]: I don't think it is necessary in this point of time krub.

**Comment by narze** [comment]
There're some undone tasks eg. translating to Thai language :)

**Pull request closed without merging**